chore: update changelog (#94)
cosmoconsult/alpaca: docs/en-us/history-roadmap/changelogs/frontend/azdevops-vscode-extension.md

All notable changes to the Visual Studio Code Extension

## 1.30.0 (2025-05-09)

> - Fix long loads of the tree, implementing pruneExpiredEntries function responsible for cleaning up expired cache entries (https://dev.azure.com/cc-ppi/Self-Service/_workitems/edit/4299/)

Reviewer: The changelog should be focused on the end user, so please avoid implementation specifics.
```suggestion
- Fix long loads of the tree views, add mechanism to clean up expired cache entries (https://dev.azure.com/cc-ppi/Self-Service/_workitems/edit/4299/)
```
